VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -70,4 +70,22 @@ public class Subscription: VIMModelObject
             "vod_rental_expiration_warning": "vodRentalExpirationWarning",
             "account_expiration_warning": "accountExpirationWarning"]
     }
+    
+    // MARK: - Helpers
+    
+    /// Helper method that determine whether a user has all the subscription settings turned off.
+    ///
+    /// - Returns: A boolean that indicates whether the user has all the settings for push notifications disabled.
+    public func areSubscriptionsDisabled() -> Bool
+    {
+        return (self.comment == false &&
+                self.credit == false &&
+                self.like == false &&
+                self.mention == false &&
+                self.reply == false &&
+                self.follow == false &&
+                self.vodPreorderAvailable == false &&
+                self.videoAvailable == false &&
+                self.share == false)
+    }
```

Comment:
@fyell - I think that's a great idea. I opened a ticket that i will work on for refactor that: https://vimean.atlassian.net/browse/VIM-5065. Because we want to add unit tests also.